### ISSUE TITLE
Instance: Violet Hold Heroic

### ISSUE THREAD

**Issue by babyjebas** [dalaranwow/dalaran-wow#661]
Issue Present: No reputation was received while killing the mobs in Violet Hold Heroic mode.

How it should work: Reputation should be given from all mobs killed for example:
http://www.wowhead.com/npc=30660/portal-guardian

**Comment by ashido96** [comment]
After Ichoron goes through his encounter mechanics three times he just stays in his pool of water at 1 health and continues to attack the group, immune to all damage and aoe

Also, the next portal will not spawn after the time goes through, the mobs should continue to spawn through even with the boss in this state.

**Comment by Akkalevil** [comment]
One year and a half later, the same bug is still not fixed...

**Comment by jonanone** [comment]
Missing reputation in heroic mode has been added.

This update will be applied to the live realm soon.

**Comment by Rushor** [comment]
jockey wrote:

Each mob gives +15 with Alliance Vanguard, bosses give nothing.
On Wowhead it said that the creatures' deaths should reward: 19 reputation with The Taunka
39 reputation with The Wyrmrest Accord (stops at Revered)
19 reputation with The Frostborn
I'm not sure if this was changed after WotLK, because wowhead isn't so accurate in this expansion. I tried looking for some information on wikiwow or openwow but nothing helped.
